ush/python/pygfs/jedi/jedi.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+
+import os
+from logging import getLogger
+from typing import List, Dict, Any, Optional
+from jcb import render
+from wxflow import (AttrDict,
+                    chdir, rm_p,
+                    parse_j2yaml,
+                    logit,
+                    Task,
+                    Executable,
+                    WorkflowException)
+
+logger = getLogger(__name__.split('.')[-1])
+
+
+class JEDI:
```

Comment:
Class names should be in CamelCase, even if it is otherwise an acronym that would be in all-caps.
```suggestion
class Jedi:
```